department-of-veterans-affairs/caseflow
# Add new file indicator

Connects #5593 

### Description
Adds the new file indicator to the Reader Link. Adds a new endpoint which currently returns all of the new files, since we think we might want more than just if there are new files in the future (this will make changes to surface what types of files easier.)

### Acceptance Criteria 
- [ ] New file icons appear when there are new files.

### Testing Plan
1. Connect to eFolder
  1. Make sure eFolder is running on port 4000
  1. Turn on the `use_efolder_locally` flag in `development.rb`
  1. Change line 37 of efolder_service to `headers = { "FILE-NUMBER" => "DEMO#{vbms_id}" }`
1. Load a queue and ensure all the files have the new file indicator.
1. Click on one make sure it goes away.
1. Add a file in the eFolder DB to that case and ensure the file indicator comes back.
